app/code/BodenkoVV/AskQuestion/Controller/Adminhtml/Question/PostDataProcessor.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+namespace BodenkoVV\AskQuestion\Controller\Adminhtml\Question;
+
+use BodenkoVV\AskQuestion\Model\DomValidationState;
+use Magento\Framework\App\Arguments\ValidationState;
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\Config\Dom\ValidationException;
+use Magento\Framework\Config\Dom\ValidationSchemaException;
+
+/**
+ * Class PostDataProcessor
+ * @package Magento\Cms\Controller\Adminhtml\Page
+ */
+class PostDataProcessor
+{
+    /**
+     * @var \Magento\Framework\Stdlib\DateTime\Filter\Date
+     */
+    protected $dateFilter;
+
+    /**
+     * @var \Magento\Framework\View\Model\Layout\Update\ValidatorFactory
+     */
+    protected $validatorFactory;
+
+    /**
+     * @var \Magento\Framework\Message\ManagerInterface
+     */
+    protected $messageManager;
+
+    /**
+     * @var DomValidationState
+     */
+    private $validationState;
+
+    /**
+     * @param \Magento\Framework\Stdlib\DateTime\Filter\Date $dateFilter
+     * @param \Magento\Framework\Message\ManagerInterface $messageManager
+     * @param \Magento\Framework\View\Model\Layout\Update\ValidatorFactory $validatorFactory
+     * @param DomValidationState $validationState
+     */
+    public function __construct(
+        \Magento\Framework\Stdlib\DateTime\Filter\Date $dateFilter,
+        \Magento\Framework\Message\ManagerInterface $messageManager,
+        \Magento\Framework\View\Model\Layout\Update\ValidatorFactory $validatorFactory,
+        DomValidationState $validationState = null
+    ) {
+        $this->dateFilter = $dateFilter;
+        $this->messageManager = $messageManager;
+        $this->validatorFactory = $validatorFactory;
+        /** @var ValidationState $this */
+        $this->validationState = $validationState
```

Comment:
Avoid using of object manager, remove validationState injection

Reply:
+